Makefile

```diff
@@ -218,8 +221,16 @@ dist/postgres.yaml: test/e2e/manifests/postgres.yaml
 	# Create Postgres e2e manifests
 	cat test/e2e/manifests/postgres.yaml | sed 's/:latest/:$(IMAGE_TAG)/' > dist/postgres.yaml
 
+local:
```

Comment:
* [ ] Test by setting kubectx to production system and run `make local`, expect exit code 1.